packages/mobile/src/screens/chat-screen/ChatScreen.tsx

```diff
@@ -325,31 +325,31 @@ export const ChatScreen = () => {
     }
   }, [earliestUnreadIndex, chatMessages])
 
-  const newestReceivedMessageId = useMemo(() => {
-    const idx = chatMessages.findIndex(
-      (chat) => chat.sender_user_id !== userIdEncoded
-    )
-    return idx >= 0 ? chatMessages[idx].message_id : null
-  }, [chatMessages, userIdEncoded])
+  const newestMessage = chatMessages.length > 0 ? chatMessages[0] : null
```

Comment:
tiny nit: i've been using `latestMessage` vs `earliestMessage`. tho tbh newer vs older maybe is easier to understand.

Reply:
`latest` is fine! I'd rather be consistent.